android/app/build.gradle

```diff
@@ -266,7 +266,7 @@ android {
 
 
 dependencies {
-    androidTestImplementation('com.wix:detox:+') {
+    androidTestImplementation('com.wix:detox-legacy:+') {
```

Comment:
I guess you have this in the PR desc, but can we also leave a comment here on why we use `legacy`?

Reply:
Added in aeeca981b7fdd864cda811e04dce48aebf0b46d6!